benchmarks/cacheGetFieldValues.mjs

```diff
@@ -0,0 +1,18 @@
+import { bench, group, run } from 'mitata'
+import { getFieldValues } from '../lib/cache/util.js'
+
+const values = [
+  'foo',
+  'invälid',
+  'foo, '
+]
```

Comment:
Can we add some more values?
I believe the ones you added for testing should be enough 👍

Reply:
Like this?

```suggestion
const values = [
  '',
  'foo',
  'invälid',
  'foo, ',
  'foo, bar',
  'foo, bar, baz',
  'foo, bar, baz, ',
  'foo, bar, baz, , '
]
```

Reply:
Done